docs/next.config.js

```diff
@@ -33,10 +33,8 @@ module.exports = {
   },
   typescript: {
     // Motivated by https://github.com/vercel/next.js/issues/7687
-    ignoreDevErrors: true,
     ignoreBuildErrors: true,
   },
-  webpack5: true,
```

Comment:
Webpack 4 was removed in Next.js v12 https://github.com/vercel/next.js/blob/37d11008250b3b87dfa4625cd228ac173d4d3563/errors/webpack5.md